apt-hook/20apt-esm-hook.conf

```diff
@@ -1,8 +1,10 @@
 APT::Update::Post-Invoke-Stats {
+	"[ ! -f /usr/lib/ubuntu-advantage/ua_update_messaging.py ] || python3 /usr/lib/ubuntu-advantage/ua_update_messaging.py";
```

Comment:
Note I would like to separate this back to a systemd timer so we don't get "heavyweight" python calls on every apt update/upgrade etc.